apps/08jquery-towers-of-hanoi/script.js

```diff
@@ -1,5 +1,42 @@
 'use strict';
+var $block = null;
 
 $(document).ready(function() {
   // Put app logic here
+  $('div[data-stack]').on('click', function() {
+    //to do
```

Comment:
TODO? Looks like you accomplished this task, so make sure you clean up the TODOs when they're done!